corehq/sql_accessors/sql_templates/get_all_cases_modified_since_3.sql

```diff
@@ -5,7 +5,7 @@ RETURNS SETOF form_processor_commcarecasesql AS $$
 BEGIN
     RETURN QUERY
     SELECT * FROM form_processor_commcarecasesql as case_table
-        WHERE (case_table.modified_on, case_table.id) > (reference_date, last_id)
+        WHERE (case_table.server_modified_on, case_table.id) > (reference_date, last_id)
```

Comment:
Couldn't figure out why the reindex was stuck in a loop and couldn't reproduce locally. Fresh eyes in the morning :eyeglasses: